src/wp-includes/blocks.php

```diff
@@ -485,7 +485,7 @@ function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
 			 *
 			 * @return string Returns the block content.
 			 */
-			$settings['render_callback'] = function( $attributes, $content, $block ) use ( $template_path ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+			$settings['render_callback'] = static function( $attributes, $content, $block ) use ( $template_path ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
Outside of the scope of this PR, but we may as well remove the `// phpcs:ignore` comment as the `VariableAnalysis` standard is not being used by Core, nor WPCS.

Isn't this also a Gutenberg file though ?

Reply:
> Isn't this also a Gutenberg file though ?

Based on the filename it would seem so, but it seems to fundamentally be a different file from https://github.com/WordPress/gutenberg/blob/trunk/lib/blocks.php

Reply:
I agree we should look into `// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable`, but not in this ticket.